js/modules/k6/http/request_test.go

```diff
@@ -485,18 +481,14 @@ func TestRequestAndBatch(t *testing.T) {
 		assert.Contains(t, err.Error(), "unsupported protocol scheme")
 
 		logEntry := hook.LastEntry()
-		if assert.NotNil(t, logEntry) {
-			assert.Equal(t, logrus.WarnLevel, logEntry.Level)
-			assert.Contains(t, logEntry.Data["error"].(error).Error(), "unsupported protocol scheme")
-			assert.Equal(t, "Request Failed", logEntry.Message)
-		}
+		assert.Nil(t, logEntry)
 
 		t.Run("throw=false", func(t *testing.T) {
 			hook := logtest.NewLocal(state.Logger)
 			defer hook.Reset()
 
 			_, err := common.RunString(rt, `
-				var res = http.request("", "", { throw: false });
+				var res = http.request("GET", "some://example.com", null, { throw: false });
 				throw new Error(res.error);
```

Comment:
I think you should `throw` some other error here and check for it below. Right now the `assert.Contains(t, err.Error(), "unsupported protocol scheme")` check below will be satisfied even if `throw` was `true`.

Reply:
hm... yeah... it wasn't working previously ... so I guess it will be nice if we now check it ... more strictly ;)